webapp/params.go

```diff
@@ -190,3 +190,30 @@ func ParsePathsParam(r *http.Request) (paths mapset.Set) {
 	}
 	return paths
 }
+
+// ParseLabelsParam returns a set list of test-run labels to include, or nil if
+// no labels are provided.
+func ParseLabelsParam(r *http.Request) (labels mapset.Set) {
+	labelParams := r.URL.Query()["label"]
```

Comment:
Do we do this "singular or plural" work elsewhere? Just thinking about the convenience/complexity tradeoff here, but that's moot if we have established a pattern already.

Reply:
Yes, this was a copy-paste of some code above.
I'll do you one better and refactor it.